roles/telegraf_client/templates/telegraf.conf.j2

```diff
@@ -27,6 +27,7 @@
 ###############################################################################
 
 [outputs]
+{% if telegraf_influxdb == true %}
```

Comment:
It seems, for bools following should be used:
```jinja2
{% if variable is sameas true %}
    variable is true
{% else %}
    variable is not true
{% endif %}
```
https://jinja.palletsprojects.com/en/2.10.x/templates/#sameas